frontend/src/_ui/RestAPIToggle/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Do we need a new component for `RestAPIToggle`? Can't we add support for `subText`/`helpText` in the existing `Toggle` component itself?

Reply:
That makes sense. @parthy007 Could you check please?